docs/root/operations/admin.rst

```diff
@@ -310,8 +310,8 @@ explanation of the output.
   .. http:get:: /stats/prometheus
 
   Outputs /stats in `Prometheus <https://prometheus.io/docs/instrumenting/exposition_formats/>`_
-  v0.0.4 format. This can be used to integrate with a Prometheus server. Currently, only counters and
-  gauges are output. Histograms will be output in a future update.
+  v0.0.4 format. This can be used to integrate with a Prometheus server. Counters, gauges and
```

Comment:
nit: I think you can delete the last sentence instead of specifically listing all of them.